__std__/flake.nix

```diff
@@ -141,6 +146,10 @@
         devShells = inputs.std.harvest inputs.self [ "plutus" "devshells" ];
         packages = inputs.std.harvest inputs.self [ "plutus" "packages" ];
       }
+      {
+        hydraJobs = inputs.std.harvest inputs.self [ "automation" "hydraJobs" ];
+        packages = inputs.std.harvest inputs.self [ "automation" "hydraJobs" ];
```

Comment:
Added as `packages` too so that we can test it locally with `nix build`
... Unless there is a way to build a `hydraJobs` flake output?
@blaggacao